.github/workflows/deploy.yml

```diff
@@ -20,24 +20,11 @@ jobs:
           distribution: temurin
           java-version: 11
       - run: sbt scio-repl/assembly
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1.1.4
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          draft: true
       - name: Upload Repl
-        uses: actions/upload-release-asset@v1.0.2
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
```

Comment:
so the token isn't needed anymore?

Reply:
passing the toke as env variable is now deprecated. The new action has a default param set to the token:
> token: Secret GitHub Personal Access Token. Defaults to ${{ github.token }}